app/services/clients/publishing_api.rb

```diff
@@ -15,20 +15,24 @@ def initialize
       @per_page = 700
     end
 
-    def content_ids
+    def all_content_items
```

Comment:
The name of this method does not represent very well what it does. I would prefer the client no to group things either. What about something like:

```ruby
def fetch_all_attributes(*attributes)
# returns array [{content_id: "id", locale: "en"},...]
end
```